codecov_auth/commands/owner/interactors/start_trial.py

```diff
@@ -1,14 +1,17 @@
 from codecov.commands.base import BaseInteractor
-from codecov.commands.exceptions import ValidationError
+from codecov.commands.exceptions import Unauthorized, ValidationError
 from codecov.db import sync_to_async
+from codecov_auth.helpers import current_user_part_of_org
 from codecov_auth.models import Owner
 from plan.service import PlanService
 
 
 class StartTrialInteractor(BaseInteractor):
```

Comment:
if we use this interactor in graphql fields, we probably can make use of the decorator `@require_part_of_org`